Lombiq.JsonEditor.Test.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -0,0 +1,142 @@
+using Atata;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using Newtonsoft.Json.Linq;
+using OpenQA.Selenium;
+using Shouldly;
+using System.Threading.Tasks;
+
+namespace Lombiq.JsonEditor.Tests.UI.Extensions;
+
+public static class TestCaseUITestContextExtensions
+{
+    private const string SampleContentItemId = "4xapn6ykttkk6wbbwgg1aaxqda";
+    private const string HelloValue = "hello";
+    private const string WorldValue = "world";
+    private const string TestField = "testField";
+    private const string TestValue = "testValue";
+
+    private static readonly By ObjectByXPath = By.XPath($"//div[@class='jsoneditor-readonly' and contains(text(),'object')]");
+    private static readonly By ObjectCountByXPath = By.XPath($"//div[@class='jsoneditor-value jsoneditor-object' and contains(text(),'{{2}}')]");
+    private static readonly By ArrayByXPath = By.XPath($"//div[@class='jsoneditor-field' and contains(text(),'printThese')]");
+    private static readonly By ArrayCountByXPath = By.XPath($"//div[@class='jsoneditor-value jsoneditor-array' and contains(text(),'[2]')]");
+    private static readonly By FieldByXPath = By.XPath($"//div[@class='jsoneditor-field' and contains(text(), '{TestField}')]");
+
+    public static async Task TestJsonEditorBehaviorAsync(this UITestContext context)
+    {
+        await context.EnableJsonEditorFeatureAsync();
+
+        await context.ExecuteJsonEditorSampleRecipeDirectlyAsync();
+
+        // Checking if the sample item is displayed correctly.
+        await context.GoToContentItemByIdAsync(SampleContentItemId);
+
+        context.Exists(By.XPath($"//div[contains(text(),'These are coming from the JSON field:')]"));
+        context.Exists(By.XPath($"//li[contains(text(),'{HelloValue}')]"));
+        context.Exists(By.XPath($"//li[contains(text(),'{WorldValue}')]"));
+
+        await context.SignInDirectlyAsync();
+        await context.GoToContentItemEditorByIdAsync(SampleContentItemId);
+
+        // Testing if input is saved.
+        context.GetAll(By.XPath($"//button[@class='jsoneditor-button jsoneditor-contextmenu-button']"))[1].Click();
```

Comment:
Can't you target the button more directly instead of relying on the matched elements' order?